index.html

```diff
@@ -80,11 +66,178 @@ <h1 class="primary-title font-family-poppins">I'm Belden.<br> Glad to see you!</
+                    Hello I'm a software developer! I can help you build a product, feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don't hesitate to contact me.
+                </p>
+                <a href="#" class="social-header font-family-poppins">LET'S CONNECT</a>
+                <ul class="social-media">
+                    <li>
+                        <a href="#" target="_blank" rel="noopener noreferrer">
+                            <i class="fa-brands fa-square-github"><span class="sr-only">github</span></i>
+                        </a>
+                    </li>
+                    <li>
+                        <a href="#" target="_blank" rel="noopener noreferrer">
+                            <i class="fa-brands fa-twitter"><span class="sr-only">twitter</span></i>
+                        </a>
+                    </li>
+                    <li>
+                        <a href="#" target="_blank" rel="noopener noreferrer">
+                            <i class="fa-brands fa-linkedin-in"><span class="sr-only">linkedin</span></i>
+                        </a>
+                    </li>
+                    <li>   
+                        <a href="#" target="_blank" rel="noopener noreferrer">
+                        <i class="fa-brands fa-whatsapp"><span class="sr-only">whatsapp</span></i>
+                        </a>
+                    </li>                    
+                </ul>
+                <button class="column-button">Get my resume</button>
+            </div>
+            
+                <ul class="features">
+                    <li>
+                        <div>
+                           Languages
+                           <ol class="feature-items">
+                                <li>
+                                    <img src="images\js.jpg" alt="">
+                                    <span>Javascript</span>
+                                </li>
+                                <li>
+                                    <img src="images\html.jpg" alt="">
+                                    <span>HTML</span>
+                                </li>
+                                <li>
+                                    <img src="images\css.jpg" alt="">
+                                    <span>CSS</span>
+                                </li>
+                            </ol>
+                        </div> 
+                        <i class="fa-solid fa-greater-than"></i>
+                       
+                    </li>
+                    <li>
+                       <div> Frameworks</div> 
+                        <i class="fa-solid fa-greater-than"></i>
+                    </li>
+                    <li>
+                        <div> Skills</div> 
+                        <i class="fa-solid fa-greater-than"></i>
+                    </li>
+                </ul>
+            
+        </section>
```

Comment:
- [ ] While you did a good job implementing all the suggested changes :+1:  I have noticed that there are some still missing. 
  - I think it would be better if you change the `background` of your button to match the template.
  - Also you should change the icon `>` for `v` like the Figma design :+1: .
  - Finally, as you may have noticed each language shouldn't;t have a border just like the suggested design provided by Microverse :+1: 

**Your design**

![Screenshot from 2023-01-14 08-31-04](https://user-images.githubusercontent.com/105117832/212474567-5b7901c9-d1de-4244-9f10-9dde5615c068.png)

**Figma design**

![Screenshot from 2023-01-14 08-37-05](https://user-images.githubusercontent.com/105117832/212474619-74caab6f-92ac-4c6c-bd88-ab87bfcd404f.png)